cmd/vclusterctl/cmd/get/service_cidr.go

```diff
@@ -0,0 +1,78 @@
+package get
+
+import (
+	"fmt"
+
+	"github.com/loft-sh/vcluster/cmd/vclusterctl/flags"
+	"github.com/loft-sh/vcluster/cmd/vclusterctl/log"
+	"github.com/loft-sh/vcluster/pkg/helm/values"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type serviceCIDRCmd struct {
+	Context   string
+	Namespace string
+	log       log.Logger
+}
+
+func getServiceCIDR(globalFlags *flags.GlobalFlags) *cobra.Command {
+	cmd := &serviceCIDRCmd{
+		Context:   globalFlags.Context,
+		Namespace: globalFlags.Namespace,
+		log:       log.GetInstance(),
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:   "service-cidr",
+		Short: "Prints Service CIDR of the cluster",
+		Long: `
+#######################################################
+############### vcluster get service-cidr  ###################
+#######################################################
+Prints Service CIDR of the cluster
+
+Ex: 
+vcluster get service-cidr
+
+Service CIDR: 10.96.0.0/12
+#######################################################
+	`,
+		RunE: func(cobraCmd *cobra.Command, args []string) error {
+			return cmd.Run(cobraCmd)
+		}}
+
+	return cobraCmd
+}
+
+func (cmd *serviceCIDRCmd) Run(cobraCmd *cobra.Command) error {
+	// first load the kube config
+	kubeClientConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(clientcmd.NewDefaultClientConfigLoadingRules(), &clientcmd.ConfigOverrides{
+		CurrentContext: cmd.Context,
+	})
+
+	// load the rest config
+	kubeConfig, err := kubeClientConfig.ClientConfig()
+	if err != nil {
+		return fmt.Errorf("there is an error loading your current kube config (%v), please make sure you have access to a kubernetes cluster and the command `kubectl get namespaces` is working", err)
+	}
+
+	client, err := kubernetes.NewForConfig(kubeConfig)
+	if err != nil {
+		return err
+	}
+
+	cidr, err := values.GetServiceCIDR(client, "default", false)
+	if err != nil {
+		return err
+	}
+
+	output := "Service CIDR: " + cidr + "\n"
```

Comment:
just print the service cidr, which makes it easier to use in an automated fashion

Reply:
done